hazelcast/src/test/java/com/hazelcast/test/AbstractHazelcastClassRunner.java

```diff
@@ -19,7 +19,6 @@
 import com.hazelcast.core.Hazelcast;
 import com.hazelcast.core.HazelcastInstance;
```

Comment:
What I often do to reduce the size of a PR, is to pull out all preparation work of a PR. These testing improvements can be very useful for everyone, so they deserve a PR of their own. It also prevents bouncing of PR's. It makes more clear what the intend is of a pr since a PR only has a single responsibility. And it makes reviewing a lot easier.